lxd/storage/drivers/driver_cephfs.go

```diff
@@ -992,6 +992,6 @@ func (d *cephfs) BackupVolume(vol Volume, targetPath string, optimized bool, sna
 	return ErrNotImplemented
 }
 
-func (d *cephfs) RestoreBackupVolume(vol Volume, snapshots []string, srcData io.ReadSeeker, op *operations.Operation) (func(vol Volume) error, func(), error) {
+func (d *cephfs) RestoreBackupVolume(vol Volume, snapshots []string, srcData io.ReadSeeker, optimized bool, op *operations.Operation) (func(vol Volume) error, func(), error) {
```

Comment:
@monstermunchkin could you call this arg `binaryFormat` to keep its meaning consistent with the `HasBinaryFormat` property it comes from in `backup.Info`?

Reply:
@tomponline The `BackupVolume` function has `optimized bool`, that's the reason I went for that name but I'll change it for `RestoreBackupVolume`.

Reply:
@monstermunchkin ah, I see, I'd be happy to have the BackupVolume argument changed to `binaryFormat` as well to keep it consistent. I think I originally picked `optimized` because it came from Backup.OptimizedStorage() function, so if we translate that into what it is, binaryFormat that is probably clearer as to what it actually means.